docs/observability/langwatch.md

```diff
@@ -0,0 +1,181 @@
+# Agent Observability with LangWatch
+
+[LangWatch](https://langwatch.ai) is a comprehensive observability platform for monitoring, debugging, and improving LLM applications and AI Agents. It provides automatic tracing, evaluation, and monitoring capabilities for your Google ADK applications through seamless OpenTelemetry integration. To get started, sign up for a [free account](https://app.langwatch.ai).
+
+## Overview
+
+LangWatch can automatically collect traces from Google ADK using [OpenInference instrumentation](https://github.com/Arize-ai/openinference/tree/main/python/instrumentation/openinference-instrumentation-google-adk), allowing you to:
+
+- **Trace agent interactions** - Automatically capture every agent run, tool call, model request, and response with context and metadata
+- **Evaluate performance** - Assess agent behavior using custom or pre-built evaluators and run experiments to test agent configurations
+- **Monitor in production** - Set up real-time dashboards and alerts to track performance
+- **Debug issues** - Analyze detailed traces to quickly identify bottlenecks, failed tool calls, and any unexpected agent behavior
+
+## Installation
+
+Install the required packages:
+
+```bash
+pip install langwatch google-adk openinference-instrumentation-google-adk
+```
+
+## Setup
+
+### 1. Configure Environment Variables { #configure-environment-variables }
+
+Set your LangWatch API key and Google API key:
+
+```bash
+export LANGWATCH_API_KEY=[your_langwatch_api_key_here]
+export GOOGLE_API_KEY=[your_google_api_key_here]
+```
+
+### 2. Connect your application to LangWatch { #connect-your-application-to-langwatch }
+
+```python
+import langwatch
+from openinference.instrumentation.google_adk import GoogleADKInstrumentor
+
+# Initialize LangWatch with the Google ADK instrumentor
+langwatch.setup(
+    instrumentors=[GoogleADKInstrumentor()]
+)
+```
+
+That's it! All Google ADK agent activity will now be traced and sent to your LangWatch dashboard automatically.
```

Comment:
Note, I was able to send a test trace to LangWatch, but I kept seeing this message at the top of the Analytics page:

> Setup pending
> Your project is not set up yet so you won't be able to see any data on the dashboard, please go to the setup page to get started.

But I couldn't figure out what other actions I needed to do at this point to go deeper into the platform or to view details of the trace that was sent.